mesa/space.py

```diff
@@ -15,28 +15,11 @@
 # pylint: disable=invalid-name
 
 import itertools
-
 import numpy as np
 
 
-def accept_tuple_argument(wrapped_function):
-    """ Decorator to allow grid methods that take a list of (x, y) coord tuples
-    to also handle a single position, by automatically wrapping tuple in
-    single-item list rather than forcing user to do it.
-
-    """
-
-    def wrapper(*args):
-        if isinstance(args[1], tuple) and len(args[1]) == 2:
-            return wrapped_function(args[0], [args[1]])
-        else:
-            return wrapped_function(*args)
-
-    return wrapper
-
-
 class Grid:
-    """ Base class for a square grid.
+    """ Class for a square grid.
```

Comment:
Shouldn't it be a rectangle grid?

Also, the doc-string for the methods below is not updated

Reply:
The height and width of the tiles are not really specified in this class, and these values are only given for visualization...so I am not sure they should be called square grid or rectangular grid. Should we introduce these new properties?